fe/fe-core/src/main/java/com/starrocks/load/ExportMgr.java

```diff
@@ -365,6 +375,10 @@ public long loadExportJob(DataInputStream dis, long checksum) throws IOException
                 newChecksum ^= jobId;
                 ExportJob job = new ExportJob();
                 job.readFields(dis);
+                // discard expired job right away
+                if (isJobExpired(job, currentTimeMs)) {
+                    LOG.warn("discard expired job: {}", job);
+                }
```

Comment:
seems only print log. Do you want to continue?

Reply:
You're right, it's a bug. I'll fix it.